tests/llmobs/test_llmobs_service.py

```diff
@@ -1617,6 +1618,24 @@ def process_trace(self, trace):
         llmobs_service.disable()
 
 
+def test_llmobs_fork_disabled(monkeypatch):
+    """Test that after being disabled the service remains disabled when forking"""
+    monkeypatch.setenv("DD_LLMOBS_ENABLED", "0")
+    svc = ddtrace.llmobs.LLMObs(tracer=DummyTracer())
```

Comment:
I think you can just use the `llmobs_service` import here instead of importing the top-level module:
```suggestion
    svc = llmobs_service(tracer=DummyTracer())
```

Reply:
ah ty